self_hosting_machinery/inference/inference_hf.py

```diff
@@ -277,6 +277,7 @@ def infer(self, request: Dict[str, Any], upload_proxy: UploadProxy, upload_proxy
                                          do_sample=temperature >= 0.05,
                                          return_dict_in_generate=True,
                                          output_scores=True,
+                                         begin_suppress_tokens=[scratchpad.eos_token],
```

Comment:
Does this fix the problem?

Reply:
I checked this fix with cases from https://github.com/smallcloudai/refact/issues/315 - it's working.